Makefile

```diff
@@ -92,3 +92,32 @@ delete-awsfederatedaccountaccess: check-aws-account-id-env
 
 .PHONY: test-awsfederatedaccountaccess
 test-awsfederatedaccountaccess: check-aws-account-id-env create-account create-awsfederatedrole create-awsfederatedaccountaccess test-switch-role delete-awsfederatedaccountaccess delete-awsfederatedrole delete-account
+
+.PHONY: deploy-aws-account-operator-credentials
+deploy-aws-account-operator-credentials:
+# Base64 Encode the AWS Credentials
+	$(eval ID=$(shell base64 <<< ${AWS_ACCESS_KEY_ID}))
+	$(eval KEY=$(shell base64 <<< ${AWS_SECRET_ACCESS_KEY}))
+
+.PHONY: deploy-aws-account-operator
+deploy-aws-account-operator:
+# Create aws-account-operator namespace
+	@oc get namespace ${NAMESPACE} || oc new-project ${NAMESPACE}
+# Create the aws-account-operator-credentials secret
+	@oc process -p AWS_ACCESS_KEY_ID=${ID} -p AWS_SECRET_ACCESS_KEY=${KEY} -f hack/templates/aws_v1alpha1_aws_account_operator_credentials.tmpl | oc apply -f -
+# Create aws-account-operator CRDs
+	@ls deploy/crds/*crd.yaml | xargs -L1 oc apply -f
+# Create zero size account pool
+	@oc apply -f - <<< '{"apiVersion": "aws.managed.openshift.io/v1alpha1", "kind": "AccountPool", "metadata": {"name": "test-account-pool"}, "spec":{"poolSize": 0}}'
+
+.PHONY: deploy
+deploy: check-aws-credentials deploy-aws-account-operator-credentials deploy-aws-account-operator
+
+.PHONY: .check-aws-credentials
```

Comment:
Should we add this to `check-aws-account-id-env`?

Reply:
I avoided doing that because the AWS ID & Key are not required for the test-awsfederatedaccountaccess target, and the AWS Account IDs are not required for the deploy target.  This way a user doesn't need to set or export them all to do one or the other.